### ISSUE TITLE
Buckets for dev testing

### ISSUE THREAD

**Issue by de-code** [elifesciences/data-pipeline#106]
It would be good to have separate buckets for dev testing the pipeline end-to-end.

**Comment by giorgiosironi** [comment]
Are these the buckets in https://github.com/elifesciences/builder/blob/master/projects/elife.yaml#L2033 ?
If so we only need to create another stack named `data-pipeline--end2end` (or similar).

**Comment by de-code** [comment]
Yes, the same buckets. There shouldn't be another instance running, otherwise there would be a race condition (it moves files it process to the archive).
